ref-impl-isy-security-rw/src/main/java/de/bund/bva/isyfact/shop/core/ProduktVerwaltung.java

```diff
@@ -0,0 +1,47 @@
+package de.bund.bva.isyfact.shop.core;
+
+import de.bund.bva.isyfact.shop.service.rest.exception.ProduktNotFoundException;
+import de.bund.bva.isyfact.shop.core.daten.ProduktBo;
+import de.bund.bva.isyfact.shop.core.impl.ProduktVerwaltungImpl;
+
+import java.util.List;
+
+/**
+ * This interface supplies facade methods for the core package.
+ * For an implementation of this interface:
+ *
+ * @see ProduktVerwaltungImpl
+ */
+public interface ProduktVerwaltung {
+
+    /**
+     * Searches for Produkt business objects with a given name.
+     * If no such name is passed:
+     * - For users in department (Abteilung) 'Zentrale':
+     *   all Product business objects are returned, without any restriction.
+     * - For all other users:
+     *   ProduktNotFoundException
+     *
+     * @param name name of products to search for
+     * @return list of products found
+     * @throws ProduktNotFoundException, if no name is given (by users NOT in 'Zentrale')
```

Comment:
if no name is given (filtering by name is required for all users NOT in 'Zentrale')